fix: mintlify docs

krushnarout opened the pull request:
fixes #2082 
/claim #2082

https://github.com/user-attachments/assets/6f0091c3-8643-46eb-85b7-7b72e07931ff

krushnarout commented:
@beastoin I've updated the docs to the latest version, fixed some issues, and added socials. Please check and let me know!

krushnarout commented:
@beastoin @kodjima33 I've updated the docs to the latest version, fixed some issues, and added socials. Please check and let me know!

beastoin commented:
1/ man, do they support import our .md file from omi repo. for example https://github.com/BasedHardware/omi/tree/main/app/README.md . why do i ask? because we don't want to duplicate the .md

@krushnarout

krushnarout commented:
> 1/ man, do they support import our .md file from omi repo. for example https://github.com/BasedHardware/omi/tree/main/app/README.md . why do i ask? because we don't want to duplicate the .md
> 

Mintlify doesn’t support directly referencing Markdown files outside its docs folder. You’ll need to duplicate them inside the docs folder.

@beastoin

kodjima33 commented:
@krushnarout can you 
1. add SDK docs? 
2. resolve conflicts? 

I'll merge in 1 hour

krushnarout commented:
> 1. add SDK docs?
> 2. resolve conflicts?
> 
> I'll merge in 1 hour

@kodjima33 Sure!

krushnarout commented:
> 
> 1. add SDK docs?
> 2. resolve conflicts?
> 
> I'll merge in 1 hour

@kodjima33 Done! Added SDK docs, resolved conflicts, and updated Import Apps doc according to the latest changes!

kodjima33 commented:
checking
